airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/TemporalJobType.java

```diff
@@ -12,5 +12,6 @@ public enum TemporalJobType {
   RESET_CONNECTION,
   CONNECTION_UPDATER,
   REPLICATE,
-  NOTIFY
+  NOTIFY,
```

Comment:
although we are doing our best to make sure connector temporal workers don't interact with platform bits until the refactor, I think it's appropriate to add a connector type here just so we have a master list. @jdpgrailsdev thoughts?

Reply:
I think that's fine, though we should probably add a comment or a line comment between the platform and connector enum values so that people know where they are currently in use.